src/bot/telegram_client.rs

```diff
@@ -1,6 +1,8 @@
 use crate::config::Config;
 use crate::http_client;
+use core::fmt::Debug;
```

Comment:
is it really needed?

Reply:
It is needed because the Debug trait symbol is inside the macro.